src/css/utilities/_spacing.scss

```diff
@@ -27,65 +27,6 @@
 }
 
 
-.u-spacing-shortest {
```

Comment:
I think these are currently commented out, but will be kept if we need them later on. Any reason for deleting them?

Reply:
These were purely duplicated rules that [were not commented out](https://github.com/a11yproject/a11yproject.com/blame/a442075085a0173abbc13b41848ff2affdf3df4c/src/css/utilities/_spacing.scss), so stylelint was complaining about them. 
Would you prefer that I leave the lines in place but comment them out?